mod-gobi/mod-gobi.postman_collection.json

```diff
@@ -121,30 +121,963 @@
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"id\": \"{{noPrivPermId}}\",\n\t\"userId\": \"{{noPrivUserId}}\",\n\t\"permissions\": []\n}"
+						},
+						"url": {
+							"raw": "{{protocol}}://{{url}}:{{okapiport}}/perms/users",
+							"protocol": "{{protocol}}",
+							"host": [
+								"{{url}}"
+							],
+							"port": "{{okapiport}}",
+							"path": [
+								"perms",
+								"users"
+							]
+						},
+						"description": "Create a user with insufficient privs to negative testing"
+					},
+					"response": []
+				},
+				{
+					"name": "/authn/login (OKAPI) - NoPrivUser",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "99aa0e9b-1d64-41b9-811c-094b4b6d419f",
+								"type": "text/javascript",
+								"exec": [
+									"pm.test(\"Status code is 201\", function () {",
+									"    pm.response.to.have.status(201);",
+									"});",
+									"",
+									"let tenant = pm.environment.get(\"xokapitenant\");",
+									"let token = postman.getResponseHeader(\"x-okapi-token\");",
+									"pm.globals.set(\"noPrivXOkapiToken\", token);"
+								]
+							}
+						},
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "d75a59b9-367d-4055-86c6-0b6c1cf9b86c",
+								"type": "text/javascript",
+								"exec": [
+									""
+								]
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "x-okapi-tenant",
+								"value": "{{xokapitenant}}"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"username\":\"akott\",\"password\":\"{{noPrivPassword}}\"}"
+						},
+						"url": {
+							"raw": "{{protocol}}://{{url}}:{{okapiport}}/authn/login",
+							"protocol": "{{protocol}}",
+							"host": [
+								"{{url}}"
+							],
+							"port": "{{okapiport}}",
+							"path": [
+								"authn",
+								"login"
+							]
+						}
+					},
+					"response": []
 				}
 			]
 		},
 		{
 			"name": "Positive Tests",
 			"item": [
 				{
-					"name": "/validate",
+					"name": "/validate",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "c6566d89-e8ea-4fca-a388-d40938eb27f0",
+								"type": "text/javascript",
+								"exec": [
+									"pm.test(\"Status code is 204\", function () {",
+									"    pm.response.to.have.status(204);",
+									"});"
+								]
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "x-okapi-tenant",
+								"value": "{{xokapitenant}}"
+							},
+							{
+								"key": "x-okapi-token",
+								"value": "{{xokapitoken}}"
+							},
+							{
+								"key": "content-type",
+								"value": "application/json"
+							},
+							{
+								"key": "Authorization",
+								"value": "apiKey SHAzQkdpVndqR19mczAwMDAwMDAwX2ZzMDAwMDAwMDA",
```

Comment:
I think this is here by mistake, mod-gobi shouldn't receive the apiKey. Even if it did, the apiKey shouldn't be hardcoded in the test (unless, perhaps, we were testing bad keys). Now that I look closer, this header was disabled but left in the test. Good ol' Postman will export everything. :(

Reply:
yup, human error.  Good catch